src/main/java/edu/tamu/modspineomatic/config/model/Okapi.java

```diff
@@ -0,0 +1,34 @@
+package edu.tamu.modspineomatic.config.model;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+
+@Component
+@ConfigurationProperties(prefix = "okapi")
+public class Okapi {
```

Comment:
If we are using Lombok, we can use Lombok annotation/s here as well. I think it is a good way to keep code to a minimum.